test/parallel/test-timers-api-refs.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+// don't verify the globals for this test
+common.globalCheck = false;
+
+// try overriding global APIs to make sure
+// they're not relied on by the timers
+global.clearTimeout = assert.fail;
```

Comment:
@Fishrock123 I've updated the test to something that is what you describe, as I understand it. PTAL